example/lib/iss_tracker.dart

```diff
@@ -0,0 +1,93 @@
+import 'dart:convert';
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:mapbox_gl/mapbox_gl.dart';
+import 'package:http/http.dart' as http;
+import 'page.dart';
+
+class IssTrackerPage extends Page {
+  IssTrackerPage()
+      : super(const Icon(Icons.settings_input_antenna), 'ISS Tracker');
+
+  @override
+  Widget build(BuildContext context) {
+    return const IssTracker();
+  }
+}
+
+class IssTracker extends StatefulWidget {
+  const IssTracker();
+
+  @override
+  State createState() => IssTrackerState();
+}
+
+class IssTrackerState extends State<IssTracker> {
+  Timer timer;
+  LatLng issPosition;
+  MapboxMapController mapController;
+  bool setInitialPosition = false;
+  Symbol _currentSymbol;
+
+  @override
+  void initState() {
+    super.initState();
+    timer = Timer.periodic(const Duration(seconds: 1), _getData);
+    _getData(null);
+  }
+
+  Future<void> _getData(Timer timer) async {
+    final http.Response data = await http.Client().get('http://api.open-notify.org/iss-now');
+    final Map<String, String> position = Map<String, String>.from(jsonDecode(data.body)['iss_position']);
+    final LatLng _issPosition = LatLng(double.parse(position['latitude']), double.parse(position['longitude']));
+
+    mapController.moveCamera(CameraUpdate.newLatLng(_issPosition));
```

Comment:
I'm sometimes getting an exception here, because mapController can still be null at this point.
I'd suggest to only call `_getData` once `onStyleLoadedCallback` has been called, to make sure that mapController is not null and that annotation operations are possible (also see #257 on that)